### ISSUE TITLE
Improve help on WebUI port, or change the port to an unprivileged port

### ISSUE THREAD

**Issue by saveriomiroddi** [danmayer/coverband#402]
Hello!

I think usage of 1022 as default port for the WebUI is problematic, since it's a privileged port on Linux.

The command as displayed in the README won't work:

```sh
$ bundle exec rake coverband:coverage_server
D, [2020-09-29T10:35:58.359745 #371296] DEBUG -- : using default configuration
Thin web server (v1.7.2 codename Bachmanity)
Maximum connections set to 1024
Listening on localhost:1022, CTRL+C to stop
rake aborted!
no acceptor (port is in use or requires root privileges)
```

Now, this is an issue, because just using `sudo` won't work, because when switching to root, the ruby/gemset may change, considering that a Rails apps generally work in the context of a specific Ruby and gemset defined by Bundler.

Now, this can be worked around using the option `COVERBAND_COVERAGE_PORT`:

```sh
$ COVERBAND_COVERAGE_PORT=8086 bundle exec rake coverband:coverage_server
D, [2020-09-29T10:36:58.979299 #372630] DEBUG -- : using default configuration
Thin web server (v1.7.2 codename Bachmanity)
Maximum connections set to 1024
Listening on localhost:8086, CTRL+C to stop
```

I therefore suggest to clarify this in the README, although I think switching port to an unprivileged one would be a simpler UX.

**Comment by danmayer** [comment]
good call, I will update the default and update the readme to call out the `COVERBAND_COVERAGE_PORT` option

**Comment by danmayer** [comment]
fixed in release 5.0.2
